tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+
+MAKE='make --debug=v --output-sync'
+
+# build and push rook image
+$MAKE build BUILD_REGISTRY=local
+build_Image="local/ceph-amd64:latest"
```

Comment:
Here as well, use either `snake_case` or `camelCase`.

Reply:
I hope `BUILD_REGISTRY` is a variable used internally with make so we cant change it